src/panels/KaitoModelsPanel.ts

```diff
@@ -96,21 +100,102 @@ export class KaitoModelsPanelDataProvider implements PanelDataProvider<"kaitoMod
     nullIsFalse(value: boolean | null): boolean {
         return value ?? false;
     }
+    parseGPU(gpuRequirement: string): [string, number] {
+        const regex = /^Standard_NC(\d+)(ads_A100_v4|s_v3)$/;
+        const match = gpuRequirement.match(regex);
+        if (match) {
+            const cpus = parseInt(match[1], 10);
+            let family: string;
+            switch (match[2]) {
+                case "ads_A100_v4":
+                    family = "StandardNCADSA100v4Family";
+                    break;
+                case "s_v3":
+                    family = "standardNCSv3Family";
+                    break;
+                default:
+                    throw new Error("Unknown gpu format");
+            }
+
+            return [family, cpus];
+        } else {
+            throw new Error("Unknown gpu format");
+        }
```

Comment:
💡 minor suggestions: either do it in separate PR for update but line-116 and line 122 both have same return set, which could be either extracted and return again, just a note only sake. (In case you notice more deeper pattern)